zegami_sdk/annotation.py

```diff
@@ -18,9 +19,8 @@ class _Annotation():
     TYPE = None
     UPLOADABLE_DESCRIPTION = None
 
-    def __init__(self, collection, annotation_data, source=None):
-        """
-        !! STOP !! Instantiate a non-hidden subclass instead.
+    def __init__(self, collection, id, row_index, source=0):
+        """!! STOP !! Instantiate a non-hidden subclass instead.
```

Comment:
Check out AbstractBaseClass which makes it impossible to instantiate the base class. https://docs.python.org/3/library/abc.html

Reply:
Solved in new commit, doesn't actually require abc as this functionality is enforced a different way